.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+python: '3.5'
+sudo: false
+env:
+  - TOXENV=py26
+  - TOXENV=py27
```

Comment:
I didn't want to add too much and kept what was already tested. I agree that python 2.6 is a bit old now and that detox should be tested against python 3.4, 3.5 and even 3.6. If all maintainers agree on this I can add these versions to be tested.